### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Liliya-V**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mporam) [inline .idea/vcs.xml]: this file needs to be in your gitignore

File: homepage.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio Liliya Voevodina.html</title>
```
Comment (mporam) [inline homepage.html]: remove the .html from the title

File: homepage.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        <link href="normalizeportfolio.css" rel="stylesheet" type="text/css">
```
Comment (mporam) [inline homepage.html]: can you name this file normalize.css - to keep it consistant

File: homepage.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="stylesheet.css" rel="stylesheet" type="text/css">
```
Comment (mporam) [inline homepage.html]: can you put all your css files into a css folder please?

File: homepage.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <div class="hometop">
+            <nav>
+                <img class="mypicture" src="IMG_9169.jpg" alt="Liliya">
```
Comment (mporam) [inline homepage.html]: can you put your images into an images folder

File: stylesheet.css [hunk 6]
```diff
@@ -0,0 +1,106 @@
+*{
```
Comment (mporam) [inline stylesheet.css]: `* {` not `*{`

File: stylesheet.css [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+body {
+    background-color :#f2f2f2;
+    font-size: 62.5%;
+    font-family: HKGrotesk, sans-serif;
+    height: 100vh;
+    width: 100vw;
+    position: relative;
+
```
Comment (mporam) [inline stylesheet.css]: remove blank line

File: stylesheet.css [hunk 8]
```diff
@@ -0,0 +1,106 @@
+*{
+    box-sizing:border-box;
+}
+
+body {
+    background-color :#f2f2f2;
```
Comment (mporam) [inline stylesheet.css]: `background-color: #f2f2f2;` not `background-color :#f2f2f2;`

File: stylesheet.css [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+    font-size: 62.5%;
+    font-family: HKGrotesk, sans-serif;
+    height: 100vh;
+    width: 100vw;
+    position: relative;
+
+}
```
Comment (mporam) [inline stylesheet.css]: missing line break between `}` and new selector

File: stylesheet.css [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+.hometop {
+    background-color: #F8D5E2;
+    height: 50vh;
+    width: 99vw;
+    padding: 10px 10px 0 10px;
+    margin: 5px 5px 0 5px;
+    position:relative;
+
```
Comment (mporam) [inline stylesheet.css]: remove line

File: stylesheet.css [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+}
+nav {
+    width: 98%;
```
Comment (mporam) [inline stylesheet.css]: why 98% and not 100%?

File: stylesheet.css [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+.hometop {
+    background-color: #F8D5E2;
+    height: 50vh;
+    width: 99vw;
+    padding: 10px 10px 0 10px;
+    margin: 5px 5px 0 5px;
+    position:relative;
+
+}
+nav {
```
Comment (mporam) [inline stylesheet.css]: missing line break before selector

File: stylesheet.css [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+    width: 98%;
+}
+.mypicture {
+    width: 100px;
+    height: 100px;
+    border: 2px solid cornflowerblue;
```
Comment (mporam) [inline stylesheet.css]: please use a hex instead of the colour name

File: stylesheet.css [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+}
+nav a {
+    text-decoration: none;
+    font-size: 2vh;
+    color: black;
+    float: right;
+    margin: 14px;
+}
+a:hover {
+    color: cornflowerblue;
```
Comment (mporam) [inline stylesheet.css]: use a hex

File: stylesheet.css [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+        display: flex;
+        flex-direction: column;
+    }
+}
+
+.homecenter {
+    /*margin: 0 auto;*/
+    width: 100%;
+    height: 90%;
+    /*display: flex;*/
```
Comment (mporam) [inline stylesheet.css]: remove any commented code

File: stylesheet.css [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+    top: 5%;
+    z-index: 2;
+}
+.arrows {
+    /*margin: auto auto;*/
+    width: 60%;
+    height: 90%;
+    position: absolute;
+    left: 20%;
+
```
Comment (mporam) [inline stylesheet.css]: remove line

File: stylesheet.css [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+h1 {
+    color: #404040;
+    font-family: "NoeText", sans-serif;
+    font-size: 6vw;
+    z-index: 3;
+    position: absolute;
+    bottom: 15px;
+    left: 25%;
+    margin: 0;
+
```
Comment (mporam) [inline stylesheet.css]: remove line

File: stylesheet.css [hunk 18]
```diff
@@ -0,0 +1,106 @@
...
+    z-index: 3;
+    position: absolute;
+    bottom: -60px;
+    left: 25%;
+    margin: 0;
+}
+.social {
+    display:flex;
+    flex-direction: column;
+
```
Comment (mporam) [inline stylesheet.css]: remove

**Review by mporam — CHANGES_REQUESTED** [review]
please make sure your css follows the following format always:

```
selector {
    property: value;
}

selector {
    property: value;
}

File: css/stylesheet.css [hunk 19]
```diff
@@ -0,0 +1,108 @@
...
+    height: 100px;
+    border: 2px solid #adddef;
+    border-radius: 50%;
+    margin: 0;
+    padding: 0;
+}
+
+nav a {
+    text-decoration: none;
+    font-size: 2vh;
```
Comment (mporam) [inline css/stylesheet.css:38]: why is this a vh value? that means it will shrink with the screen, is that what you want?

**Review by mporam — CHANGES_REQUESTED** [review]
you need to update your html with the new CSS and image locations, as they dont currently work

File: css/stylesheet.css (hunk 19 above)
Comment (Liliya-V) [reply css/stylesheet.css:38]: Done it on purpose so my text is responsive

**Review by mporam — APPROVED** [review]

**Pull request merged**